contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
Yeah if yo look at what is in  the CentOS vagrant file setup I just had to have the username and password in extra args. But that meant I did not have those same variables set in the `group_vars` I left them commented out.